plugins/frontend/backstage-plugin-iframe/src/components/IFrameComponent.tsx

```diff
@@ -26,20 +27,14 @@ import { configApiRef, useApi } from '@backstage/core-plugin-api';
 import { determineError } from './utils/helpers';
 import { useEntity } from '@backstage/plugin-catalog-react';
 
-type IFrameComponentContentProps = {
-  src: string;
-  title: string;
-  height?: string;
-  width?: string;
-};
-
 const IFrameComponentContent = (props: IFrameComponentContentProps) => {
-  const { title, src, height, width } = props;
+  const { class: className, height, src, title, width } = props;
```

Comment:
Note: `class` is a valid property name, but an invalid variable name due to it being a reserved keyword.

Reply:
can we name this `classes` or `extraClasses` instead of using reserved keywords (`class` in JS, `className` in React)

Reply:
`className` is not a reserved word and it makes it explicit to where this will be used was my thinking. I can rename this to whatever though.

Reply:
Sorry I meant the incoming prop in here. We shouldn't be using the `class` keyword as a prop name since it's confusing with JSX JS class and HTML class attribute naming convention mixups.

Reply:
I switched it to using `classes` instead.